TradeItIosEmsApi/TradeItConnector.m

```diff
@@ -337,7 +339,8 @@ - (NSArray *)getLinkedLogins {
 }
 
 - (void)unlinkLogin:(TradeItLinkedLogin *)login
-          localOnly:(BOOL)localOnly {
+          localOnly:(BOOL)localOnly
```

Comment:
It has been added recently by alex:
https://github.com/tradingticket/TradeItIosTicketSDK2/commit/d0a11f08255cd4d42371b4228cc4715cc1c4a9cb#diff-aeb438906a8062e23d46cc61ab15c791